packages/app/src/components/nominationBanner/nominationBanner.js

```diff
@@ -1,51 +1,80 @@
-import style from './style.css'
+import style from './style.css';
 import React from 'react';
 const states = require('us-state-codes');
 
 const NominationBanner = ({ nomination }) => {
-  const date = new Date(nomination.dateReceived).toLocaleDateString()
-  const lastName = nomination.patientName ? nomination.patientName.split(' ')[1] : ''
-  const state = states.getStateCodeByStateName(nomination.hospitalState)
-  const nominationName = `${lastName}-${state}`
-  const formattedAmount = nomination.amountRequestedCents ? (nomination.amountRequestedCents).toFixed(2).replace(/\d(?=(\d{3})+\.)/g, '$&,') : ''
+  const date = new Date(nomination.dateReceived).toLocaleDateString();
+  const lastName = nomination.patientName ? nomination.patientName.split(' ')[1] : '';
+  const state = states.getStateCodeByStateName(nomination.hospitalState);
+  const nominationName = `${lastName}-${state}`;
+  const formattedAmount = nomination.amountRequestedCents ? nomination.amountRequestedCents.toFixed(2).replace(/\d(?=(\d{3})+\.)/g, '$&,') : '';
+
+
+  const handleDecline = () => {};
```

Comment:
Is this function supposed to do something?

Reply:
forgot to delete this! Thought I was adding functionality to this button but that's for another ticket.